themes/light_spinel.json

```
@@ -21,7 +21,9 @@
     "activityBar.activeBackground": "#d8d8db",
     "activityBar.activeBorder": "#acabb3",
     "list.activeSelectionBackground": "#cdccd3",
+    "list.inactiveSelectionBackground": "#cdccd3",
```

Let's use this slightly lighter grey that we use in other places to give a bit of contrast between when the item is selected vs not.

```suggestion
    "list.inactiveSelectionBackground": "#d8d8db",
```